index.html

```diff
@@ -2554,42 +2556,52 @@ <h3>
             The user agent is free to display the icon in any context.
           </dd>
         </dl>
+        <p class="note">
+          If an icon contains multiple purposes, it can be used for any of
+          those purposes. If all of the stated purposes are not recognized, the
+          icon is totally ignored. For example, if an icon has purpose
+          <code>"badge fizzbuzz"</code>, then it may be used as a badge, but if
+          an icon has purpose <code>"fizzbuzz"</code>, then it must be ignored.
```

Comment:
This note is sounding very much like conformance requirements - which are already stated above. It should be phased like it's describing behavior to a developer, not giving conformance requirements to an implementer.

Reply:
Fixed. I assume this is basically asking to change phrasing like "the user agent must" to "the user agent will", basically describing to a developer what would happen in certain situations assuming a conformant user agent, rather than describing conformance. Also avoiding RFC 2119 words.

Reply:
Also, I removed the final sentence "This allows new purposes to be added to the standard in the future without them being interpreted as normal icons by older user agents." which is really more an explanation of _why_ we made it work this way, neither requirements nor an explanation of _how_ it works, so I took that out. Unless you think this is still useful?

Reply:
No, I think it's good now... I should write a ReSpec validation rule to maybe suggest alternative to  RFC 2119 keywords in Notes.